### PR TITLE
PGB-1237 BRP REST api uitbreiden met adres

### PR DESCRIPTION
**Pull request by Vegter**
This PR contains all changes that were requested during the last reviews.

**Introduction**

BRP is complex, not at code level but at MKS interpretation level.
The PR has been open for some time and has been updated in the meantime with new results of the reviews.
The effect is that the PR has become (too) big in number of files changed.
In order to not increase the complexity of this PR any new work will now be postponed until this PR has been reviewed.

**Contents**

- Use a code resolver to resolve gemeente and land codes
Gemeente code are resolved using the GOB API for BRK Gemeentes
Land codes are resolved by the inclusion of a csv-file from the BZK government site
 
- Included a 500 Internal Server Error response that conforms with the haalcentraal API
 
- Improved the response conversion by allowing literal values
 
- Solve birthdate issues, including the use of a XML attribute indicator_incomplete_date
 
- Include links to BAG API when a nummeraanduiding is found
 
**Main changes**
The main changes are in base_response and mks_utils and the new code resolver class

### CODE REVIEW

File: src/gobstuf/mks_utils.py [hunk 1]
```diff
@@ -47,37 +127,106 @@ def _dd(cls, mks_datum):
...
+        """
+        Age calculation that takes leap years into account
+
+        :param now:
+        :param birthday:
+        :return:
+        """
+        try:
+            day = birthday.replace(year=now.year)
+        except ValueError:
```
Comment (JKoetsier) [inline src/gobstuf/mks_utils.py:170]: Bit of a wide exception.

Using calendar.isleapyear() with check for month and day is more explicit and cleaner I think.

Comment (Vegter) [reply src/gobstuf/mks_utils.py:170]: Nice remark! Will check on it

File: src/gobstuf/mks_utils.py [hunk 2]
```diff
@@ -47,37 +127,106 @@ def _dd(cls, mks_datum):
...
+            return None
+
         if cls._is_mks_datum(mks_geboortedatum):
             # Interpret all dates as dates in the current timezone
             now = _today()
-            birthday = datetime.datetime.strptime(mks_geboortedatum, cls._MKS_DATUM_PARSE_FORMAT)
-            return relativedelta(now, birthday).years
+            birthday = datetime.datetime.strptime(mks_geboortedatum, cls._MKS_DATUM_PARSE_FORMAT).date()
+            day_is_unknown = not incomplete_date_indicator.is_dag_bekend()
+            if not (day_is_unknown and now.month == birthday.month):
```
Comment (JKoetsier) [inline src/gobstuf/mks_utils.py:203]: Lot of negating going on here. Not really clear at first sight what's meant here.

File: src/gobstuf/mks_utils.py (hunk 1 above)
Comment (Vegter) [reply src/gobstuf/mks_utils.py]: Much easier to read now

File: src/gobstuf/mks_utils.py (hunk 2 above)
Comment (Vegter) [reply src/gobstuf/mks_utils.py]: And so for this one

**Comment by Vegter** [comment]
Rewrote age calculation and changed method names to english while leaving attribute names in dutch

**Pull request merged**